Cargo.toml

```diff
@@ -103,15 +106,15 @@ tui = [
 ]
 
 [patch.crates-io]
-equihash = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-pczt = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-transparent = { package = "zcash_transparent", git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_address = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_client_backend = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_client_sqlite = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_encoding = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_keys = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_primitives = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_proofs = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zcash_protocol = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
-zip321 = { git = "https://github.com/zcash/librustzcash.git", rev = "62e35406edb008e82d6245760f68942d363afc1c" }
+equihash = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+pczt = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+transparent = { package = "zcash_transparent", git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_address = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_client_backend = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_client_sqlite = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_encoding = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_keys = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_primitives = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_proofs = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zcash_protocol = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
+zip321 = { git = "https://github.com/zcash/librustzcash.git", rev = "0489ed87940fc729963edb3cc5d59985b52b3326" }
```

Comment:
This should be updated to zcash/librustzcash@41362ab0530fd256d2d27ca8361b36f9b3734f2c which fixes bugs in the 0.16 migration.